components/common-go/log/log.go

```diff
@@ -52,8 +52,7 @@ func Init(service, version string, json, verbose bool) {
 		log.SetFormatter(&gcpFormatter{
 			log.JSONFormatter{
 				FieldMap: log.FieldMap{
-					log.FieldKeyLevel: "severity",
```

Comment:
There is no way to map the values using the JSONFormatter, so we need to remove this mapping, as it overwrites otherwise.